src/main/java/seedu/address/model/person/Person.java

```diff
@@ -19,21 +19,31 @@ public class Person {
     // Identity fields
     private final Name name;
     private final Phone phone;
+    private final Income income;
     private final Email email;
+    private final Address address;
+    private final Family family;
+    //To remove phone
+
+    //To remove email
+
 
     // Data fields
-    private final Address address;
+
     private final Set<Tag> tags = new HashSet<>();
 
     /**
      * Every field must be present and not null.
      */
-    public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tags) {
+    public Person(Name name, Phone phone, Income income, Email email, Address address, Family family, Set<Tag> tags) {
         requireAllNonNull(name, phone, email, address, tags);
```

Comment:
Do we need to include income and family as non-null too?

Reply:
👍